lms/static/sass/views/_program-details.scss

```diff
@@ -260,6 +260,11 @@ $btn-color-primary: $primary-dark;
     }
   }
 
+  .tab-pane {
+    .program-details-content {
+       padding-left: 40px;
+     }
+   }
   .course-list-heading {
```

Comment:
You added a new SCSS rather than updating the existing one, can you explain that a little bit? 
https://github.com/edx/edx-platform/blob/bf64c34d9de678d6ee289641a03c3d6af435f88d/lms/static/sass/views/_program-details.scss#L237-L247

Reply:
It is added here to apply only to tabbed view, not on the older one.